src/_P036_FrameOLED.ino

```diff
@@ -425,7 +425,10 @@ boolean Plugin_036(uint8_t function, struct EventStruct *event, String& string)
 
       if (CONFIG_PIN3 != -1) // Button related setup
       {
-        pinMode(CONFIG_PIN3, INPUT_PULLUP);
+#ifndef ESP32
+        // pinMode can not be used for ESP32, must be set on the hardware page
+        pinMode(CONFIG_PIN3, INPUT_PULLUP);     //  Reset pinstate
+#endif
```

Comment:
Hmm this is new to me.
If this indeed is a limitation on ESP32, please make sure to explain it in the documentation.

Reply:
According to the ESP32 datasheet each input can be configured as pulldown (45kOhm), pullup (45kOhm) or high impedance.
With the standard GPIO setting within the plug in I have no chance to set pulldown as I need on my device. Therefore I had to use the hardware page to define the input setting of the GPIO used in the plug in.
In future it would be nice to do it again within the plug in settings.